sync/src/main/java/tech/pegasys/teku/sync/SyncManager.java

```diff
@@ -227,6 +231,13 @@ private boolean isPeerSyncSuitable(Eth2Peer peer) {
         network.getPeerCount(),
         ourFinalizedEpoch.toString(10));
     return !peersWithSyncErrors.contains(peer.getId())
-        && peer.getStatus().getFinalizedEpoch().compareTo(ourFinalizedEpoch) > 0;
+        && (peer.getStatus().getFinalizedEpoch().compareTo(ourFinalizedEpoch) > 0
+            || isMoreThanAnEpochAhead(peer));
+  }
+
+  private boolean isMoreThanAnEpochAhead(final Eth2Peer peer) {
```

Comment:
nit: to be more clear
```suggestion
  private boolean isHeadMoreThanAnEpochAhead(final Eth2Peer peer) {
```